src/helpers/parser.ts

```diff
@@ -6,18 +6,22 @@ import {
   type StatementType,
 } from "@tableland/sqlparser";
 import { isTestnet } from "./chains.js";
+import { type NameMapping } from "./config.js";
 
 export type { NormalizedStatement, StatementType };
 
-export async function normalize(sql: string): Promise<NormalizedStatement> {
+export async function normalize(
+  sql: string,
+  nameMap?: NameMapping
```

Comment:
The wasm parser already has `nameMap` as an optional second argument. Just need to add it here for TS to work.